dialogue-annotations-processor/src/test/resources/com/palantir/myservice/service/MyServiceDialogueServiceFactory.java.generated

```diff
@@ -194,11 +191,9 @@ public final class MyServiceDialogueServiceFactory implements DialogueServiceFac
                 if (header5.isPresent()) {
                     _request.putHeaderParams("h5", _parameterSerializer.serializeDouble(header5.getAsDouble()));
                 }
-                _request.putAllHeaderParams(
-                        "h6",
-                        header6.stream()
-                                .map(_parameterSerializer::serializeString)
-                                .collect(Collectors.toList()));
```

Comment:
nit: Can we avoid changing this from `collect` to `forEach` in this PR? It's probably not a meaningful difference in practice, but it's probably marginally better to put parameters in bulk. If we have reason to believe that switching to `forEach` would be an improvement, let's make that change in a separate PR with the requisite justification/benchmarks/etc.

Reply:
Good point. I mostly did it because it was not meaningfully different and because conjure-java uses a for loop & a `put` instead of `putAll` https://github.com/palantir/conjure-java/blob/0e8ad1430736965cd4c926ad6bcd1ff25dd06e54/conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/DefaultStaticFactoryMethodGenerator.java#L380-L388

I update the code to be `putAll` instead